tests/test-rest-post-meta-fields.php

```diff
@@ -230,6 +230,37 @@ public function test_set_value() {
 	}
 
 	/**
+	 * @depends test_get_value
+	 */
+	public function test_set_duplicate_single_value() {
```

Comment:
@joehoyle `update_metadata()` only does the duplicate check for single meta keys, for multiples it just goes ahead and performs the update.